### PR TITLE
Modifying TARGET_DB_PASSWORD in env if already present while export data from target otherwise appending it

### PR DESCRIPTION
**Pull request by ShivanshGahlot**

### CODE REVIEW

File: yb-voyager/cmd/importData.go [hunk 1]
```diff
@@ -177,7 +177,18 @@ func startExportDataFromTargetIfRequired() {
...
+		if strings.HasPrefix(envVar, "TARGET_DB_PASSWORD=") {
+			// Change the value of TARGET_DB_PASSWORD
+			env[i] = "TARGET_DB_PASSWORD=" + tconf.Password
+			found = true
+			break
+		}
+	}
+	if !found {
+		env = append(env, "TARGET_DB_PASSWORD="+tconf.Password)
+	}
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/importData.go:191]: Can you please test this for `SOURCE_DB_PASSWORD` and `SOURCE_REPLICA_DB_PASSWORD`, we might have to add this at all places as I believe we are not handling this at other places.

Comment (ShivanshGahlot) [reply yb-voyager/cmd/importData.go:191]: Added similar code for `SOURCE_DB_PASSWORD`. Couldn't find any code where `SOURCE_REPLICA_DB_PASSWORD` is being appended to the env variable.

File: yb-voyager/cmd/exportData.go [hunk 2]
```diff
@@ -654,7 +654,17 @@ func startFallBackSetupIfRequired() {
...
+			found = true
+			break
+		}
+	}
+	if !found {
+		env = append(env, fmt.Sprintf("SOURCE_DB_PASSWORD=%s", source.Password))
+	}
 	execErr := syscall.Exec(binary, cmd, env)
 	if execErr != nil {
 		utils.ErrExit("failed to run yb-voyager import data to target - %w\n Please re-run with command :\n%s", err, cmdStr)
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/exportData.go:670]: `yb-voyager import data to source` int error msg.

**Review by priyanshi-yb — APPROVED** [review]
LGTM

File: yb-voyager/cmd/exportData.go [hunk 3]
```diff
@@ -654,7 +654,17 @@ func startFallBackSetupIfRequired() {
...
+	for i, e := range env {
+		if strings.HasPrefix(e, "SOURCE_DB_PASSWORD=") {
+			env[i] = "SOURCE_DB_PASSWORD=" + source.Password
+			found = true
+			break
+		}
+	}
+	if !found {
+		env = append(env, fmt.Sprintf("SOURCE_DB_PASSWORD=%s", source.Password))
+	}
```
[the remark this answers is not in the document]
Comment (ShivanshGahlot) [reply yb-voyager/cmd/exportData.go]: Done

**Pull request merged**